src/group_theory/order_of_element.lean

```diff
@@ -293,6 +293,11 @@ lemma add_order_of_dvd_iff_nsmul_eq_zero {n : ℕ} : add_order_of x ∣ n ↔ n
 lemma order_of_dvd_iff_pow_eq_one {n : ℕ} : order_of a ∣ n ↔ a ^ n = 1 :=
 ⟨λ h, by rw [pow_eq_mod_order_of, nat.mod_eq_zero_of_dvd h, pow_zero], order_of_dvd_of_pow_eq_one⟩
 
+lemma commute.order_of_mul_dvd_lcm (H : commute a b) :
```

Comment:
Is there a reason why you use uppercase H for a hypothesis here? Especially in a file that setimws uses H for a group I would try to avoid it.

Reply:
Ah, it's because `a` and `b` used to be `g` and `h`.